spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/TableWriter.scala

```diff
@@ -67,8 +69,22 @@ class TableWriter[T] private (
     val (primaryKey, regularColumns) = columns.partition(_.isPrimaryKeyColumn)
     val (counterColumns, nonCounterColumns) = regularColumns.partition(_.isCounterColumn)
 
+    val nameToBehvaior = columnSelector collect {
+        case cn:CollectionColumnName => cn.columnName -> cn.collectionBehavior
+      } toMap
```

Comment:
don't use suffix notation if you don't really need to; generally suffix notation is discouraged, because it may cause problems with end-of-line inference